SmartDeviceLink/SDLResponseDispatcher.m

```diff
@@ -108,6 +108,10 @@ - (void)storeRequest:(SDLRPCRequest *)request handler:(nullable SDLResponseHandl
 }
 
 - (void)clear {
+    for (SDLRPCCorrelationId *key in self.rpcResponseHandlerMap) {
```

Comment:
Please add a comment explaining that we need to call all of the completion handlers and give a reason.